localisation/wc_culture_buildings.csv

```diff
@@ -1,212 +1,212 @@
+ca_culture_kobold_1;Kobold Mine;;Kobold-Mine;;;;;;;;;;;x
+ca_culture_kobold_2;Kobold Mine;;Kobold-Mine;;;;;;;;;;;x
+ca_culture_kobold_3;Kobold Mine;;Kobold-Mine;;;;;;;;;;;x
+ca_culture_kobold_4;Kobold Mine;;Kobold-Mine;;;;;;;;;;;x
+ca_culture_kobold_desc;Cowardly, dim, and weak, the rat-like kobolds stay in their mines away from civilization, digging for minerals to trade with harpies or goblins. Despite their desire to stay underground, they see very poorly in the dark, causing them to wear candles atop their heads. This need has caused them to develop a cultural attachment to their candles, with some tribes offering leadership to whoever's candle stands the tallest.;;Feig, d�mmlich und schwach, bleiben die ratten�hnlichen Kobolde in ihren Minen fernab der Zivilisation und graben nach Mineralien, um mit Harpyien oder Goblins Handel zu treiben. Trotz ihres Wunsches, unter der Erde zu bleiben, sehen sie im Dunkeln sehr schlecht, weshalb sie Kerzen auf dem Kopf tragen. Diese Not hat sie dazu veranlasst, eine kulturelle Bindung zu ihren Kerzen zu entwickeln, wobei einige St�mme demjenigen die F�hrung anbieten, dessen Kerze am h�chsten steht.;;;;;;;;;;;x
+ca_culture_furbolg_1;Totem of Ursoc and Ursol;;Totem von Ursol und Ursoc;;;;;;;;;;;x
+ca_culture_furbolg_2;Totem of Ursoc and Ursol;;Totem von Ursol und Ursoc;;;;;;;;;;;x
+ca_culture_furbolg_3;Totem of Ursoc and Ursol;;Totem von Ursol und Ursoc;;;;;;;;;;;x
+ca_culture_furbolg_4;Totem of Ursoc and Ursol;;Totem von Ursol und Ursoc;;;;;;;;;;;x
+ca_culture_furbolg_desc;The furbolg find themselves to be descendants of the wild god ancients Ursoc and Ursol. For some furbolgs however, reverence is not enough, and they choose to embody the might and ferocity of their gods, growing large and feral as they infuse themselves with the ancients' power. These are known as ursa totemics, and other furbolgs hold them in nearly as high regard as their shamans, recognizing their role for their tribes in protecting and hunting.;;Die Furbolgs sehen sich als Nachkommen der wilden G�tter Ursoc und Ursol. F�r einige Furbolgs ist Ehrfurcht jedoch nicht genug, und sie entscheiden sich daf�r, die Macht und Wildheit ihrer G�tter zu repr�sentieren, indem sie gro� und wild werden, w�hrend sie sich mit der Macht der Halbg�tter durchdringen. Diese sind als Ursa-Totemiker bekannt, und andere Furbolgs sch�tzen sie fast so hoch wie ihre Schamanen, da sie ihre Rolle f�r ihre St�mme beim Schutz und bei der Jagd erkennen.;;;;;;;;;;;x
```

Comment:
Changed find to believe.
```suggestion
ca_culture_furbolg_desc;The furbolg believe themselves to be descendants of the wild god ancients Ursoc and Ursol. For some furbolgs however, reverence is not enough, and they choose to embody the might and ferocity of their gods, growing large and feral as they infuse themselves with the ancients' power. These are known as ursa totemics, and other furbolgs hold them in nearly as high regard as their shamans, recognizing their role for their tribes in protecting and hunting.;;Die Furbolgs sehen sich als Nachkommen der wilden G�tter Ursoc und Ursol. F�r einige Furbolgs ist Ehrfurcht jedoch nicht genug, und sie entscheiden sich daf�r, die Macht und Wildheit ihrer G�tter zu repr�sentieren, indem sie gro� und wild werden, w�hrend sie sich mit der Macht der Halbg�tter durchdringen. Diese sind als Ursa-Totemiker bekannt, und andere Furbolgs sch�tzen sie fast so hoch wie ihre Schamanen, da sie ihre Rolle f�r ihre St�mme beim Schutz und bei der Jagd erkennen.;;;;;;;;;;;x
```

Reply:
according to deepL it makes sense with "find themselves"...

Reply:
Fair. Hadn't heard it used that way before, but I've found a few sources agreeing, so its fine..